pkgs/by-name/gi/git-worktree-switcher/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  makeWrapper,
+  installShellFiles,
+  git,
+  jq,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Consider using finalAttrs

Reply:
I'll look into this. I've basically modified the `thefuck` package to get this as I am not 100% familiar with all the options.

Reply:
Fixed in 7c2dda4abd2b5cef40276badf4f57dd391bc209f